aspnet-core/src/HRMv2.Core/Manager/BaseManager.cs

```diff
@@ -116,5 +119,33 @@ public string GetSurNameByFullName(string fullName)
         {
             return fullName.Substring(0, fullName.LastIndexOf(" "));
         }
+
+        public void CheckViewOrViewMyBranchEmployee(string permission)
```

Comment:
ko cần tách thành hàm này public void CheckViewOrViewMyBranchEmployee(string permission)
        {
            if (IsGranted(permission)) throw new AbpAuthorizationException($"Required permissions are not granted. [{permission}]");
        }

Reply:
vâng, hàm này em quên chưa xoá ạ